middleware/context_test.go

```diff
@@ -11,6 +11,8 @@ import (
 	"strings"
 	"testing"
 	"time"
+
+	"html/template"
```

Comment:
Is html/template intentional or did you mean text/template? We use text/template in the context.go file.